src/WebKit.cmake

```diff
@@ -11,11 +11,13 @@ set(WEBKIT_CMAKE_ARGS
     -DCMAKE_SYSTEM_PROCESSOR=arm
     -DCMAKE_XCODE_ATTRIBUTE_SUPPORTED_PLATFORMS=${CMAKE_XCODE_ATTRIBUTE_SUPPORTED_PLATFORMS}
     -DCMAKE_XCODE_EFFECTIVE_PLATFORMS=${CMAKE_XCODE_EFFECTIVE_PLATFORMS}
+    -DCMAKE_XCODE_ATTRIBUTE_IPHONEOS_DEPLOYMENT_TARGET=${CMAKE_XCODE_ATTRIBUTE_IPHONEOS_DEPLOYMENT_TARGET}
```

Comment:
Why are we adding these additional flags?

Reply:
`CMAKE_XCODE_ATTRIBUTE_IPHONEOS_DEPLOYMENT_TARGET` is necessary in order to force JSC to match the deployment target of NativeScript. Without it, it defaulted to latest (e.g. 12.4 with Xcode 10.2.1) and that resulted in compiler or maybe linker errors.